lib/omniauth/strategies/timetree.rb

```diff
@@ -24,6 +24,13 @@ def raw_info
         @raw_info = access_token.get(endpoint, options).parsed
       end
 
+      def extract_uid
+        return unless raw_info
+        return unless raw_info['data']
+
+        raw_info['data']['id']
```

Comment:
![nits](https://img.shields.io/badge/review-nits-green.svg)
I think using `dig` will be more simple.

```suggestion
        raw_info.dig('data', 'id')
```